sdf/1.12/joint_state.sdf

```diff
@@ -9,11 +9,101 @@
     <description>Name of the joint</description>
   </attribute>
 
-  <element name="angle" type="double" default="0" required="+">
+  <element name="angle" type="double" default="0" required="-1">
     <attribute name="axis" type="unsigned int" default="0" required="1">
       <description>Index of the axis.</description>
     </attribute>
 
     <description>Angle of an axis</description>
   </element>
+
+  <element name="axis_state" required="0">
+    <description>
+      Contains the state of the first joint axis.
+    </description>
+
+    <element name="position" type="double" default="0" required="0">
+      <description>The position of the first joint axis.</description>
+
+      <attribute name="degrees" type="bool" default="false" required="0">
+        <description>
+          If true, the joint position is expressed in units of degrees [deg],
+          otherwise it is expressed in radians [rad].
+        </description>
+      </attribute>
+    </element>
+
+    <element name="velocity" type="double" default="0" required="0">
+      <description>The velocity of the first joint axis.</description>
+
+      <attribute name="degrees" type="bool" default="false" required="0">
+        <description>
+          If true, the joint velocity is expressed in units of degrees per
```

Comment:
Similar to the position, can you add a qualifier for prismatic joints?

Reply:
https://github.com/gazebosim/sdformat/pull/1461/commits/1b770452e68eaf9115737d3e8a844616352d8ee3